index.html

```diff
@@ -374,48 +343,12 @@ <h5 class="card-title">John Smith</h5>
 
     <footer>
       <div class="footer-content">
-        <h3>Copyright</h3>
-
-        <p>
-          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam vitae
-          felis scelerisque, gravida sapien non, cursus augue. Aenean id pretium
-          turpis. Suspendisse eros nunc, sollicitudin nec.
-        </p>
-
```

Comment:
- [ ] Good job adding the partners @bekjon-me, but while doing so, please notice your design needs to look like the one on the provided design. 
- [ ] As mentioned by the previous reviewer, there is a partner section with a list of partners, and on the footer, a logo aligned to the left and text aligned to the right. Please make sure to have both sections.

### Given design 
![219145314-35f51028-eb6c-482f-a95c-2e0c9703b71c](https://user-images.githubusercontent.com/69077061/219753506-31a6864f-43ac-43e8-9ffd-4c7c4d17005d.png)

### Your design as of now
![image](https://user-images.githubusercontent.com/69077061/219753759-3bf1a3ef-c4db-44aa-98da-ec0a5fc4f202.png)

Reply:
- [ ] You seem to have removed this section completely, and this can only take you farther away from matching the requirement. I suggest that you take a deep breath, have a 5 minutes walk, and come sit for another coding session. :restroom: